config/migrations/2023/20112111524300-fix-typo-bedienaarfinancieringcode.sparql

```diff
@@ -0,0 +1,17 @@
+DELETE {
+  graph ?g {
+    ?instance rdf:type ?oldClass
+  }
+}
+INSERT {
+  graph ?g{
+    ?instance rdf:type ?newClass
+  }
+}
+where { 
+  graph ?g{
+    BIND (<http://lblod.data.gift/vocabularies/organisatie/BedienaarFinanceringCode> as ?oldClass)
+    BIND (<http://lblod.data.gift/vocabularies/organisatie/BedienaarFinancieringCode> as  ?newClass)
+   ?instance rdf:type ?oldClass . 
+  }
+}
```

Comment:
In DEV this migration will also cause changes in the `http://redpencil.data.gift/id/deltas/producer/organizations` graph. From what I understand, it should be avoided to apply migrations to producer or consumer graphs. Unless @claire-lovisa can confirm that in this case it is okay.